### DISCUSSION TITLE
I cannot start the android application. "Error: Could not find or load main class "

### DISCUSSION THREAD

**Discussion by kaheikleung** [WhatsApp/stickers#913] [General]
I cannot start the android application. 
I don't know which java is main class. 
Who can help me? 
![Screenshot 2022-12-02 at 1 09 53 AM](https://user-images.githubusercontent.com/95486557/205116212-f1ca07ee-6619-4865-bbbb-bc013c520131.png)
Thank you.

**Comment by Mementomori2023** [comment]
contactagent96@gmail.com 

Bonsoir  mr @Zandor300 

Mementomori123
